srcpkgs/sbs/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'sbs'
+pkgname=sbs
+version=1.0.0
+revision=1
+build_style=gnu-makefile
+make_use_env=yes
+hostmakedepends="pkg-config"
+makedepends="imlib2-devel libXinerama-devel"
+depends="imlib2-devel libXinerama-devel"
+short_desc="single purpose non-bloat background setter"
+maintainer="Onur Ozkan <work@onurozkan.dev>"
+license="GPL-2.0-only"
+homepage="https://github.com/ozkanonur/sbs"
+changelog="https://raw.githubusercontent.com/ozkanonur/sbs/master/ChangeLog"
+distfiles="${homepage}/archive/v${version}.tar.gz"
```

Comment:
don't use homepage here

Reply:
Sorry, I couldn't get what's wrong with it. Shouldn't it be the repository link?

Reply:
expand the variable to its value

Reply:
Ah, got you now I was on completely different point xd